R/utility_spliceHelper_functions.R

```diff
@@ -1,598 +1,572 @@
-# Note: several of the functions in this file are adopted from 
-# the GenomicAlignments package (Author: Hervé Pagès, Valerie Obenchain,
-# Martin Morgan)
-# License Artistic-2.0
+# Note: several of the functions in this file are adopted from the GenomicAlignments package (Author: Hervé Pagès, Valerie Obenchain, Martin Morgan)
```

Comment:
why this change? Wouldn't it be better to use the original line wrapped version for Bioconductor compatibility?

Reply:
I have reverted this back to the GoekeLab/bambu-bioc-dev version.